src/rpc/rpcversion.h

```diff
@@ -0,0 +1,22 @@
+#ifndef DEFI_RPCVERSION_H
```

Comment:
I don't think we need semver for RPC. It's just going to be numeric 1, 2, 3..

I think we can remove this entirely and just hard code both min, and current to 1 for now.

Reply:
Okay